public/app/features/dashboard/dashgrid/DashboardGrid.tsx

```diff
@@ -64,8 +64,9 @@ function GridWrapper({
   /*
     Disable draggable if mobile device, solving an issue with unintentionally
      moving panels. https://github.com/grafana/grafana/issues/18497
+     theme.breakpoints.md = 769
   */
-  const draggable = width <= 420 ? false : isDraggable;
+  const draggable = width <= 769 ? false : isDraggable;
```

Comment:
Could `theme.breakpoints.md` be referred to or does this have to be hardcoded here?

Reply:
sadly theme.breakpoints.md cannot be used as it's a string, and do not want to parse it here. We should change most variables in theme to be numbers so they can more easily be used in code